recipes/fermipy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.10.4" %}
+
+package:
+  name: fermipy
+  version: {{version}}
+
+source:
+  fn: fermipy-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/f/fermipy/fermipy-{{ version }}.tar.gz 
+  md5: 3eee87e3d2f324fdc51afe03f9d086a5
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
```

Comment:
It is fine to work on the Windows support later in the feedstock once this is merged.